### ISSUE TITLE
Allow user to turn off automatic (hidden) parallel domain decomposition

### ISSUE THREAD

**Issue by aowen87** [aowen87/BAR#404]
Some of our readers provided automatic parallel domain decomposition (Netcdf)

It might be good to provide a mechanism (Read/open option, global preference) that would allow user to control when/how this happens.
Always, Never, Auto.  With 'Auto' also providing a settable threshold at which it happens (Netcdf's current threshold is 100,000 points).

This came up because of different behavior of the IndexSelect operator in serial/parallel due to two things: decomposition, and the 'wrap' option for IndexSelect not working for multiblock data (data is single block, but decomposed when run in parallel).

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Kathleen Biagas
Original creation: 05/15/2015 02:17 pm
Original update: 05/19/2015 07:11 pm
Ticket number: 2271
